src/tests/ftest/server/daos_server_dump.yaml

```diff
@@ -0,0 +1,15 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+    - server-F
+timeout: 360
```

Comment:
Does this test really take anywhere near this long?

Reply:
hmmm, it possibly comes from the existing yaml I have used as a template. Will use 140s like for dynamic_start_stop which should b similar in terms of normal duration...